### PR TITLE
added image & input

### PR DESCRIPTION
**Pull request by sharmayatendra**

### CODE REVIEW

File: components/Image/image.html [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+            </p>
+            <p class="main-description"><span>Amaze-UI</span> has different kinds of images.</p>
+            <div class="comp-container">
+                <h2 class="comp-heading">Responsive image</h2>
+                <div class="img-container">
+                    <img src="/images/avatar-2.jpg" class="img-res">
+                </div>
+                <iframe class="code-snippet"
+  src="https://carbon.now.sh/embed?bg=rgba%287%2C94%2C84%2C1%29&t=night-owl&wt=none&l=htmlmixed&width=617&ds=false&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=14px&lh=171%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522comp-container%2522%253E%250A%2520%2520%2520%253Ch2%2520class%253D%2522comp-heading%2522%253EResponsive%2520image%253C%252Fh2%253E%250A%2520%2520%2520%2520%2520%253Cdiv%2520class%253D%2522img-container%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%253Cimg%2520src%253D%2522%252Fimages%252Favatar-2.jpg%2522%2520class%253D%2522img-res%2522%253E%250A%2520%2520%2520%2520%2520%253C%252Fdiv%253E%250A%253C%252Fdiv%253E"
+
```
Comment (devdeadviz) [inline components/Image/image.html]: Use proper code formatting.

File: components/Image/image.html [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+            </div>
+
+            <div class="comp-container">
+                <h2 class="comp-heading">Rounded image</h2>
+                <div class="img-container">
+                    <img src="/images/avatar-2.jpg" class="img-res rounded-img">
+                </div>
+                <iframe class="code-snippet"
+  src="https://carbon.now.sh/embed?bg=rgba%287%2C94%2C84%2C1%29&t=night-owl&wt=none&l=htmlmixed&width=617&ds=false&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=14px&lh=171%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522comp-container%2522%253E%250A%2520%2520%2520%253Ch2%2520class%253D%2522comp-heading%2522%253EResponsive%2520image%253C%252Fh2%253E%250A%2520%2520%2520%2520%2520%253Cdiv%2520class%253D%2522img-container%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%253Cimg%2520src%253D%2522%252Fimages%252Favatar-2.jpg%2522%2520class%253D%2522img-res%2520rounded-img%2522%253E%250A%2520%2520%2520%2520%2520%253C%252Fdiv%253E%250A%253C%252Fdiv%253E"
+  sandbox="allow-scripts allow-same-origin">
```
Comment (devdeadviz) [inline components/Image/image.html]: same here

File: components/Input/input.css [hunk 3]
```diff
@@ -0,0 +1,39 @@
+:root {
+    --error-color: rgb(255, 0, 0);
+    --success-color: rgb(0, 128, 0);
+}
```
Comment (devdeadviz) [inline components/Input/input.css]: You can create a common variable style sheet for all the variables.

File: components/Input/input.html [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+                <h2 class="comp-heading">Textbox Input</h2>
+                <div class="inp-container">
+                    <label for="inp">Label</label>
+                    <input type="text" placeholder="enter input here" required>
+                </div>
+                <iframe class="code-snippet"
+  src="https://carbon.now.sh/embed?bg=rgba%287%2C94%2C84%2C1%29&t=night-owl&wt=none&l=htmlmixed&width=617&ds=false&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=14px&lh=171%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522comp-container%2522%253E%250A%2520%2520%2520%253Ch2%2520class%253D%2522comp-heading%2522%253ETextbox%2520Input%253C%252Fh2%253E%250A%2520%2520%2520%2520%2520%253Cdiv%2520class%253D%2522inp-container%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%253Clabel%2520for%253D%2522inp%2522%253ELabel%253C%252Flabel%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%253Cinput%2520type%253D%2522text%2522%2520placeholder%253D%2522enter%2520input%2520here%2522%2520required%253E%250A%2520%2520%2520%2520%2520%253C%252Fdiv%253E%250A%253C%252Fdiv%253E"
+  
+  sandbox="allow-scripts allow-same-origin">
```
Comment (devdeadviz) [inline components/Input/input.html]: use proper code formatting

**Review by devdeadviz — CHANGES_REQUESTED** [review]
Added some required changes.

**Review by xii-vi — APPROVED** [review]
Work on the comments. Rest is fine to merge.

**Pull request merged**